apps/ex_nvr_web/lib/ex_nvr_web/live/user_list_live.ex

```diff
@@ -0,0 +1,134 @@
+defmodule ExNVRWeb.UserListLive do
+  @moduledoc false
+
+  use ExNVRWeb, :live_view
+
+  alias ExNVR.Accounts
+
+  def render(assigns) do
+    ~H"""
+    <div class="grow">
+      <div :if={@current_user.role == :admin} class="ml-4 sm:ml-0">
+        <.link href={~p"/users/new"}>
+          <.button>Add User</.button>
+        </.link>
+      </div>
+
+      <.table id="users" rows={@users}>
+        <:col :let={user} label="Id"><%= user.id %></:col>
+        <:col :let={user} label="First Name"><%= user.first_name %></:col>
+        <:col :let={user} label="Last Name"><%= user.last_name %></:col>
+        <:col :let={user} label="Username"><%= user.username %></:col>
+        <:col :let={user} label="Email"><%= user.email %></:col>
+        <:col :let={user} label="Role">
+          <div class="flex items-center">
+            <%= String.upcase(to_string(user.role)) %>
+          </div>
+        </:col>
+        <:col :let={user} label="Language">
+          <div class="flex items-center">
+            <%= String.upcase(to_string(user.language)) %>
+          </div>
+        </:col>
+        <:action :let={user}>
+          <.button
+            :if={@current_user.role == :admin}
+            id={"dropdownMenuIconButton_#{user.id}"}
+            data-dropdown-toggle={"dropdownDots_#{user.id}"}
+            class="text-sm ml-3 hover:bg-gray-100 dark:bg-gray-800"
+          >
+            <svg
+              class="w-5 h-5"
+              aria-hidden="true"
+              fill="currentColor"
+              viewBox="0 0 20 20"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M10 6a2 2 0 110-4 2 2 0 010 4zM10 12a2 2 0 110-4 2 2 0 010 4zM10 18a2 2 0 110-4 2 2 0 010 4z" />
+            </svg>
+            <div
```

Comment:
Try to use `<.modal>` component and also let's follow the same convention for ids. Separate words with `-` and not `_`.

Reply:
Do we need a popup? this is a dropdown menu button !

Reply:
I mean the popup of the delete user